From a849b23e7eef3633f5e3fe70388c19454e565128 Mon Sep 17 00:00:00 2001
From: Vasyl Gello <vasek.gello@gmail.com>
Date: Tue, 2 Jun 2020 04:44:19 +0000
Subject: [PATCH] Bump Security String to 2020-06-05

Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2020-0115   A-150038428     EoP     High            8.0, 8.1, 9, 10
CVE-2020-0117   A-151155194     RCE     Critical        8.0, 8.1, 9, 10

Previously Implemented:
============
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2020-8597   A-151153886     RCE     Critical        8.0, 8.1, 9, 10

Not Implemented:
================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
None

Not Applicable (platform source):
=================================
CVE:            References:     Type:   Severity:       Updated AOSP versions:
CVE-2019-2219   A-119041698     ID      High            8.0, 8.1, 9, 10
CVE-2019-9460   A-62535446      EoP     High            10
CVE-2020-0113   A-150944913     ID      High            9, 10
CVE-2020-0114   A-147606347     EoP     High            10
CVE-2020-0116   A-151330809     ID      High            10
CVE-2020-0118   A-150904694     EoP     High            10
CVE-2020-0119   A-150500247     ID      High            10
CVE-2020-0121   A-148180766     ID      High            10

Signed-off-by: Vasyl Gello <vasek.gello@gmail.com>
Change-Id: I63d25b01f8659330fb1a6e34b22105eaee6e8c89
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 94caf22679..975c48d874 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2020-05-05
+    PLATFORM_SECURITY_PATCH := 2020-06-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.17.1

